summoner-cli/src/Summoner/GhcVer.hs

```diff
@@ -28,6 +31,12 @@ data GhcVer
     | Ghc865
     deriving (Eq, Ord, Show, Enum, Bounded)
 
+showGhcOutput :: GhcVer -> Text
+showGhcOutput ghcVer = intercalateMap " " ($ ghcVer) [ showGhcVer
+                                                     , ("base-" <>) . baseVer
+                                                     , ("lts-" <>) . latestLts
+                                                     ]
```

Comment:
Let's change the definition of this function in the following way:

1. Write it according to our style guide, i.e.

    ```haskell
    showGhcOutpu ghcVer = someFunction
        [ ...
        , ...
        ]
    ```
2. Instead of partially applying `$` operator, let's write the usage explicitly. The code will be more readable and we have only 3 elements in this list, no need to go fancy here :slightly_smiling_face: